cg_hermes/config/balsamic.py

```diff
@@ -1,627 +1,487 @@
-        "used_by": ["audit"],
+        "used_by": ["deliver"],
     },
-    frozenset({"annotated-germline-vcf-summary", "manta-germline", "vcf-summary", "sv"}): {
-        "tags": ["manta", "germline", "vcf-report"],
+    frozenset(TAGS["vardict.vcf.gz.tbi"]): {
+        "tags": ["vardict", "vcf-snv-research-index"],
         "is_mandatory": False,
-        "used_by": ["audit"],
+        "used_by": ["deliver"],
     },
-    frozenset({"cram", "tumor-cram"}): {
-        "tags": ["cram", "tumor"],
+    frozenset(TAGS["vardict.all.filtered.pass.vcf.gz"]): {
+        "tags": ["vardict", "vcf-snv-clinical"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"cram", "tumor-cram-index"}): {
-        "tags": ["cram-index", "tumor"],
+    frozenset(TAGS["vardict.all.filtered.pass.vcf.gz.tbi"]): {
+        "tags": ["vardict", "vcf-snv-clinical-index"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"vcf-filtered", "snv", "clinical-vcf-filtered"}): {
-        "tags": ["vcf", "vcf-snv-filtered", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["storage"],
-    },
-    frozenset({"clinical-vcf-filtered-index", "vcf-filtered", "snv"}): {
-        "tags": ["vcf-index", "vcf-snv-filtered-index", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["storage"],
-    },
-    frozenset({"vcf-pass", "snv", "clinical-vcf-pass"}): {
-        "tags": ["vcf", "vcf-snv-clinical", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["scout", "deliver"],
-    },
-    frozenset({"clinical-vcf-pass-index", "vcf-pass", "snv"}): {
-        "tags": ["vcf-index", "vcf-snv-clinical-index", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["scout", "deliver"],
-    },
-    frozenset({"vcf-sv-pass", "sv", "clinical-vcf-sv-pass"}): {  # manta filtered PANEL
-        "tags": ["vcf-sv-clinical", "manta", "filtered"],
+    # TNscope_umi
+    frozenset(TAGS["TNscope_umi.vcf.gz"]): {
+        "tags": ["tnscope-umi", "vcf"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver"],
     },
-    frozenset({"clinical-vcf-sv-pass-index", "vcf-sv-pass", "sv"}): {
-        "tags": ["vcf-sv-clinical-index", "manta", "filtered"],
+    frozenset(TAGS["TNscope_umi.vcf.gz.tbi"]): {
+        "tags": ["tnscope-umi", "vcf-index"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver"],
     },
-    frozenset({"vcf-sv-pass", "clinical-vcf-sv-pass"}): {  # manta filtered WGS
-        "tags": ["vcf-sv-clinical", "manta", "filtered"],
+    frozenset(TAGS["TNscope_umi.all.filtered.pass.vcf.gz"]): {
+        "tags": ["tnscope-umi", "vcf-snv-research"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver"],
```

Comment:
```suggestion
        "used_by": ["deliver", "scout?"],
```
Same as before.. for the specific customer, these files need to be uploaded to scout.

Reply:
Updated!

Do you know on which tab this will be displayed in Scout? For panel we are already using `Clinical SNV and INDELs` for `vardict` for example

Reply:
At the moment, when customers request UMI workflow analysis and deliverables to scout. We are replacing the 
`tumor(normal).merged.cram` with `tumor(normal)_umi.merged.cram` files for IGV visualization. And for VCF we are using the same tab `Clinical SNV and INDELS` for uploading TNscope UMI VCF files instead of `vardict vcf file`.
And all this is done manually.

Reply:
Good to know! Thanks!